src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -79,23 +86,48 @@ class ReportsPageRanking extends React.Component {
               </small>
 
               {pages.map((item, index) => (
-                <div key={index} className="page-ranking--item">
-                  <p className="text-ranking">
-                    <strong>{index + 1}</strong>
-                  </p>
-                  <a
-                    className="link-ranking"
-                    href={item.url}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {item.name}
-                  </a>
-                  <p className="text-ranking">
-                    <strong>{item.totalVisitors}</strong>{' '}
-                    <FormattedMessage id="reports_pageranking.total_visits" />
-                  </p>
-                </div>
+                <PageRankingItem key={index}>
+                  <div>
```

Comment:
What do you think about moving this `<div>` element inside `PageRankingItem` or `PageRankingItemText` element?